vault/changelog.release.2022-05-17.md

```diff
@@ -0,0 +1,84 @@
+---
+id: o674526ctxg6rt8bc144rdz
+title: 0.95
+desc: ''
+updated: 1652809279352
+created: 1652755065976
+---
+
+Dendron 0.95 has sprouted  🌱
+
+**Local Configuration:** We are releasing an early build of Config Overrides, which allows a local configuration to override values in the default configuration. This lets you use your own settings and vaults within a shared team workspace.
+
+- More information here about [[local configurations|dendron://dendron.dendron-site/dendron.topic.local-config-override]]
+
+**Custom Themes:** You can now customize the style of your sites published with Dendron. Step right up, bring your own custom CSS theme, and be amazed with how good your knowledge can look!
+
+- Learn more about [[custom themes|dendron://dendron.dendron-site/dendron.topic.theme.publish.md]]
+
+**Initialization Speed-Up:** But wait, there's more: less waiting! This release drastically improves workspace initialization speed (10x faster for large workspaces 🚀 ).
+
+## Highlights
+- feat(chore): preview support for local configuration overrides
+- feat(publish): custom theme support for publishing 
+- enhance(workspace): speed up workspace initialization (up to 10x speed up of workspace initialization)
+
+## Everything Else
+- feat(views): allow customization of tree view display and sorting behavior
+- enhance(views): set `full graph` as default
+- enhance(views): display workspace information in full graph view
+- enhance(workspace): new user tutorials
+- fix(publish): published images missing leading forward slashes
+
+## Community
+
+### General Updates
+**Self Contained Vaults:**  [Self contained vaults](https://blog.dendron.so/notes/g98s3m6jft4gnb546y7i4xg/) make it easier to sync and share your notes with others. They are backwards compatible with existing vaults which means you can experiment with them without breaking your current flow. Starting next week, we will be flipping the defaults so that newly created vaults will be self contained by default. If you would like to preserve the current behavior, you can manually set [[dendron.enableSelfContainedVaultWorkspace|dendron://dendron.dendron-site/dendron.ref.config.vscode-config#dendronenableselfcontainedvaultworkspace]] to false
```

Comment:
on second thought, can we remove `If you would like to preserve the current behavior, you can manually set [[dendron.enableSelfContainedVaultWorkspace|dendron://dendron.dendron-site/dendron.ref.config.vscode-config#dendronenableselfcontainedvaultworkspace]] to false`?